registration/forms.py

```diff
@@ -247,7 +247,6 @@ def create_user(self, application, commit=True):
         user = User(
             username=self.cleaned_data['username'],
             email=self.cleaned_data['email'],
-            is_active=False,
```

Comment:
@antoviaque I really dislike the way that edX uses this field, and it's caused me a lot of problems when working on the third party auth code in their codebase.

`is_active` is a standard django field that has one purpose: [To allow "soft deletion" of users](http://stackoverflow.com/questions/29742845/django-how-do-i-use-is-active-of-auth-user-table). The `is_active` field should always be True unless you want a user to essentially be "deleted", in which case you can set it False to prevent that user from ever logging in again (but preserving the history of their activity on the website).

If we want to track email validation, I would strongly prefer to use a proper "is_email_verified" attribute on a custom User model, or use something like [this](https://github.com/mfogel/django-simple-email-confirmation/blob/ec88991/simple_email_confirmation/models.py#L203-L219), which is even better since it can properly handle changes of email address without losing historical validation information.